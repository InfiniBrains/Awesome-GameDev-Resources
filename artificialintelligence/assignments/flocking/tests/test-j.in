0.000 10.000 1.000 0.000 0.000 1.000 0.000 2
0.010 0.000 0.000 0.000
-0.010 0.000 0.000 0.000
0.125
0.125
0.125
0.25
