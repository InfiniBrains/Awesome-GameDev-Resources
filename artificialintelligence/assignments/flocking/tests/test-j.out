0.791 0.000 6.250 0.000
-0.791 0.000 -6.250 0.000
1.582 0.000 6.329 0.000
-1.582 0.000 -6.329 0.000
2.378 0.000 6.368 0.000
-2.378 0.000 -6.368 0.000
3.984 0.000 6.421 0.000
-3.984 0.000 -6.421 0.000
